src/main/java/com/base/BaseScheduledFlow.java

```diff
@@ -0,0 +1,187 @@
+package com.base;
+
+import com.getbase.Client;
+import com.getbase.Configuration;
+import com.getbase.models.*;
+import com.getbase.services.DealsService;
+import com.getbase.services.StagesService;
+import com.getbase.sync.Meta;
+import com.getbase.sync.Sync;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.MDC;
+import org.springframework.core.env.MissingRequiredPropertiesException;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Created by przemek on 19.10.2016.
+ */
+
+@Service
```

Comment:
This class can be annotated as @Component. However, annotating it as @Service is fine.